DreamJournal/DreamJournal/NewEntryView.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  ContentView.swift
+//  DreamJournal
+//
+//  Created by CUBS Customer on 10/7/22.
+//
+
+import SwiftUI
+import CoreData
+
+struct CustomColor {
```

Comment:
I love the abstraction for your colors rather than duplicating them everywhere!

I usually like to do this with an extension to Color directly, which lets you use the colors without defining the type when Swift knows that it's somewhere a Color belongs.

So you can replace `struct CustomColor` with `extension Color`, and then call your colors like `.Navy`, just like you can use the system-included `.red` elsewhere. Also I guess I'd recommend starting those with a lowercase letter, as that's more conventional.

Reply:
Oh wow, I didn't know that you could do that with colors! I don't know much about extensions, but that seems really useful. I'll have to look into changing that!